src/kixi/event_replay/types.clj

```diff
@@ -27,3 +27,13 @@
 
 (def datehour
   (s/conformer datehour? identity))
+
+(defn str-integer?
+  [x]
+  (if (string? x)
+    (Integer/parseInt x)
```

Comment:
Datastore str->int conformer catches Exception so this should do the same: 
```
(defn str->int
  [^String s]
  (try
    (Integer/valueOf s)
    (catch NumberFormatException e
      (or
       (str-double->int s)
       ::s/invalid))))
```

Reply:
cheers. It's like we cld do with some sort of central repository for these things..